IOMC/EventVertexGenerators/python/VtxSmearedParameters_cfi.py

```diff
@@ -1058,6 +1058,19 @@
     Z0 = cms.double(0.131175)
 )
 
+# From 2023 PbPb data run 374719 
+Realistic2023PbPbCollisionVtxSmearingParameters = cms.PSet(
```

Comment:
not a showstopper for the PR, but after this is merged, it would  be nice to generate a corresponding `SimBeamSpotObject` payload and upload it to the conditions database (see https://github.com/cms-sw/cmssw/pull/43197) @francescobrivio FYI

Reply:
Hi everyone, 
I have produced (locally) the corresponding payload and plotted it with the Payload Ispector:
![image](https://github.com/cms-sw/cmssw/assets/7822641/baf7b6ea-8aee-44ff-9697-22755a3bb874)

@claytoniousfunk @mandrenguyen from the plot you can see that, given the `Emittance = cms.double(2.055e-08)` value you'll get a beamspot transverse width of about `7.2 um` (from simplified calculation adapted from [LPC-LumiCalc](https://lpc.web.cern.ch/lumiCalc.html)) is this the expected value?

If yes I can proceed with uploading a the new tag to CondDB.

Reply:
No, the beamspot width should be `12.1 um`.  I thought I input everything correctly, but maybe I made a mistake.  Here is what I used in the lumi calculator:

![Screenshot from 2023-11-09 10-08-26](https://github.com/cms-sw/cmssw/assets/61713310/c8213b17-77cc-4390-aa17-e6124fdd50ab)

Reply:
Ah got it!
You used as emittance the "Normailised transverse emittance", while you should use the "Transverse emittance" (from the lower section of the LumiCalc). 
Just replace:
```
   Emittance = cms.double(2.055e-08),
```
with
```
   Emittance = cms.double(5.854e-08),
```

and you get a correct BS transverse width of 12.09:
![image](https://github.com/cms-sw/cmssw/assets/7822641/ffd7e727-73d1-42dd-99df-69209b5cce08)